src/components/SpaceCreateLegacyOsnap.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
```

Comment:
I moved the old osnap code that still relies on safesnap to a separate component so as not to clutter the main component. the logic is unchanged, but I renamed functions and variables to contain 'legacy' as to avoid confusion